brave/browser-laptop
# Adds some additional banners

# This PR switches promotions to the production server!!

Please test this PR with `LEDGER_ENVIRONMENT=staging npm run start`, so that you will still be using staging.

Submitter Checklist:


Fixed recovery promotional removal
Removes staging flag
Fixes fake visits

Resolves #12131
Resolves #12098
Resolves #11394

Auditors:

Test Plan:

a) Regular promotion
1) enable wallet
2) wait for promotion to appear
3) click on claim
4) make sure that success message appears

b) Network problem
1) enable wallet
2) wait for promotion to appear
3) disable internet
3) click on claim
4) make sure that general error message appears



Reviewer Checklist:

Tests
